src/checkTargetMatchToPR.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+const semverDiff = require('semver/functions/diff')
+
+const { targetOptions, semanticVersionOrder } = require('./getTargetInput')
+
+const expression = /from ([^\s]+) to (.+)$/
+
+const checkTargetMatchToPR = (prTitle, target) => {
+  if (target === targetOptions.major) {
+    return true
+  }
+  const match = expression.exec(prTitle)
+
+  if (match === null) {
+    return true
+  }
+  const prTarget = semverDiff(match[1], match[2])
+  if (prTarget === null) {
+    return true
+  }
+  if (
+    semanticVersionOrder.indexOf(prTarget) >
+    semanticVersionOrder.indexOf(target)
+  ) {
+    return false
+  }
+  return true
```

Comment:
this logic can be simplified with a single return statement: !diff || target <= diff